iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -3759,6 +3794,9 @@
 							};
 						};
 					};
+					ED097E752BB80C320006ED01 = {
+						LastSwiftMigration = 1530;
+					};
 				};
```

Comment:
I think this autogenerated code can be removed.

Reply:
Ah sorry I don't know how this got generated. I removed all lines.